src/main/java/io/github/thebusybiscuit/slimefun4/core/networks/cargo/CargoNet.java

```diff
@@ -12,6 +12,7 @@
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 
+import io.github.thebusybiscuit.slimefun4.implementation.items.cargo.CargoConnectorNode;
```

Comment:
Your import order needs changing to match conventions (not just this instance but all)

Reply:
k mb

Reply:
done